cmd/ocm/create/cluster/cmd.go

```diff
@@ -52,6 +52,12 @@ const (
 	gcpTermsAgreementNonInteractiveError = "Review and accept Google Terms and Agreements on " +
 		gcpTermsAgreementsHyperlink + ". Set the flag --marketplace-gcp-terms to true " +
 		"once agreed in order to proceed further."
+
+	privateFlag            = "private"
+	vpcNameFlag            = "vpc-name"
+	controlPlaneSubnetFlag = "compute-subnet"
+	computePlaneSubnetFlag = "control-plane-subnet"
```

Comment:
```suggestion
	controlPlaneSubnetFlag = "control-plane-subnet"
	computePlaneSubnetFlag = "compute-subnet"
```

Reply:
sorry, I missed amending my commit before pushing the changes